editor/src/clj/editor/tile_map.clj

```diff
@@ -1139,9 +1139,12 @@
   (output palette-renderables pass/RenderData produce-palette-renderables)
   (output renderables pass/RenderData :cached produce-tool-renderables)
   (output input-handler Runnable :cached (g/constantly (make-input-handler)))
-  (output info-text g/Str (g/fnk [cursor-world-pos tile-dimensions]
-                            (when-some [[x y] (get-current-tile cursor-world-pos tile-dimensions)]
-                              (format "Cell: %d, %d" (+ 1 x) (+ 1 y))))))
+  (output info-text g/Str (g/fnk [cursor-world-pos tile-dimensions mode palette-tile]
+                            (case mode 
+                              :editor  (when-some [[x y] (get-current-tile cursor-world-pos tile-dimensions)] 
+                                         (format "Cell: %d, %d" (+ 1 x) (+ 1 y)))
+                              :palette (when palette-tile 
+                                         (format "Tile: %d" (+ 1 palette-tile)))))))
```

Comment:
not sure if it's optimal, because i see tht we for some reason don't use `current-tile` instead we call `get-current-tile` , But maybe it's a start point and you can help me to figure out how to move on

Reply:
Yes, we could be using the `current-tile` output like this:
```clj
(when-some [[x y] current-tile]
  ...)
```
Not really sure why we didn't already do that, but `get-current-tile` doesn't look very expensive so this is fine too. 😅